torch/__init__.py

```diff
@@ -1489,3 +1489,10 @@ def _sparse_coo_tensor_unsafe(*args, **kwargs):
                   'use torch.sparse_coo_tensor(..., check_invariants=False) instead.')
     kwargs['check_invariants'] = False
     return torch.sparse_coo_tensor(*args, **kwargs)
+
+# Trigger registration of functional distributed collectives
+# This is needed by torch_deploy
+try:
+    import torch.distributed._functional_ops as funops
```

Comment:
do you want to bind these to a name (funops seems a bit playful) or do you just want to `import torch.distributed._functional_ops` here and not expose them under a short name

Reply:
I'll `del funops` to ensure nothing gets imported into the `torch` module..